### PR TITLE
Just a few spelling fixes. 

### PR DESCRIPTION
**Pull request by skoomasteve**
### Prerequisites

- [ ] I have [created an issue](https://github.com/umbraco/Umbraco-CMS/issues) for the proposed changes in this PR, the link is:
- [ ] I have added steps to test this contribution in the description below

### CODE REVIEW

File: build.bat [hunk 1]
```diff
@@ -9,6 +9,6 @@ IF ERRORLEVEL 1 (
 
 :error
 ECHO.
-ECHO Can not run build\build.ps1.
-ECHO If this is due to a SecurityError then please refer to BUILD.md for help!
+ECHO Cannot run build\build.ps1.
+ECHO If this is due to a security error then please refer to BUILD.md for help!
```
Comment (nul800sebastiaan) [inline build.bat]:
```suggestion
ECHO If this is due to a SecurityError then please refer to BUILD.md for help!
```

`SecurityError` is the error code, which should be preserved here.

**Comment by abjerner** [comment]
@skoomasteve thanks for taking the time to fix this - and congrats on your first PR to Umbraco 🎉

Besides the error code as mentioned by @nul800sebastiaan, I think your changes look good 👍

**Comment by skoomasteve** [comment]
Thank you guys, sounds good to me; I had failed to notice the existence of that error code.

**Comment by nul800sebastiaan** [comment]
Thanks @skoomasteve ! Do you have an account on Our Umbraco? I'd love to give you the contributor badge. Just link to your public profile on Our and I'll get it done!

**Comment by skoomasteve** [comment]
No problem at all @nul800sebastiaan.  I just signed up; public profile is here -->  https://our.umbraco.com/member/275666

Thanks.

**Comment by nul800sebastiaan** [comment]
Hi there @skoomasteve,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**